core/src/main/scala/kafka/server/BrokerToControllerChannelManager.scala

```diff
@@ -169,11 +182,19 @@ class BrokerToControllerRequestThread(
 ) extends InterBrokerSendThread(threadName, networkClient, config.controllerSocketTimeoutMs, time, isInterruptible = false) {
 
   private val requestQueue = new LinkedBlockingDeque[BrokerToControllerQueueItem]()
-  private var activeController: Option[Node] = None
+  private val activeController = new AtomicReference[Option[Node]](None)
```

Comment:
nit: an atomic reference of Option is a little strange. Could we just use null and change the code to use `Option(activeController.get())`?